nuxeo-services/nuxeo-platform-web-common/src/main/java/org/nuxeo/ecm/platform/ui/web/auth/NuxeoAuthenticationFilter.java

```diff
@@ -940,6 +940,9 @@ protected void initUnAuthenticatedURLPrefix() {
         unAuthenticatedURLPrefix = new ArrayList<>();
         for (String pluginName : service.getAuthChain()) {
             NuxeoAuthenticationPlugin plugin = service.getPlugin(pluginName);
+            if (plugin == null) {
+                log.error("Could not find plugin for name '" + pluginName + "'");
```

Comment:
Then `continue` to avoid the NPE on the next line, at least we'll now have a log and won't crash.